src/databricks/labs/ucx/source_code/used_table.py

```diff
@@ -15,15 +15,7 @@
 
 class UsedTablesCrawler(CrawlerBase[UsedTable]):
 
-    @classmethod
-    def for_paths(cls, backend: SqlBackend, schema) -> UsedTablesCrawler:
-        return UsedTablesCrawler(backend, schema, "used_tables_in_paths")
-
-    @classmethod
-    def for_queries(cls, backend: SqlBackend, schema) -> UsedTablesCrawler:
-        return UsedTablesCrawler(backend, schema, "used_tables_in_queries")
-
-    def __init__(self, backend: SqlBackend, schema: str, table: str):
+    def __init__(self, backend: SqlBackend, schema: str, table: str) -> None:
```

Comment:
That's bad. For clarity class factory methods should always come before instance methods